### DISCUSSION TITLE
Poor caption quality with low audio level

### DISCUSSION THREAD

**Discussion by ian-oz** [deepgram/community#847] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Streaming

### Details

Compared to other STT solutions (e.g. Azure and Google), Deepgram seems very sensitive to audio level. At low audio levels where other solutions continue to produce consistently good captions, the Deepgram WER jumps up markedly. Perhaps there needs to be server-based AVC/limiter pre-processing of audio prior to the main STT processing, or if this already exists, perhaps it needs some further work. This is currently preventing us migrating 1000's of clients to Deepgram.

Using Node.js @deepgram/sdk model=nova-2 rate=8000 encoding=linear16 lang=en-AU
Have not yet tried phonecall or voicemail variants. It would be useful to know if there is any different pre-processing done with these selections.

Thanks in advance.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @ian-oz, thanks for sharing this feedback. I'd recommend that you do try our `phonecall` model. It's very popular with customers who have audio that is lower quality or more challenging. It's not that we pre-process the audio differently, but that the model itself is trained on data from different domains and qualities.

**Reply by ian-oz** [reply]
Thanks, we'll try it.
